### PR TITLE
Removed extra finalizer

### PR DESCRIPTION
**Pull request by sshveta**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit f37c80ab27a12cc13198e778dbb69d64791ada97:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 01b90a547712e41f6b0e0f600c9f83752d7bb2ea:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/tests/services/test_provision_stack.py [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 
 pytestmark = [
+    pytest.mark.usefixtures("logged_in"),
```
Comment (mfalesni) [inline cfme/tests/services/test_provision_stack.py]: I guess this should not be required. Pete is removing all the logged_in references and logged_in fixture anyway.

**Comment by dajoRH** [comment]
Lint report for commit d66219f0b990f7db188cbc14dc7dbe1a8c961ddd:

Everything seems all right :smile: .
*CFME QE Bot*

**Pull request merged**